docgen/templates/annex.md

```diff
@@ -2,7 +2,7 @@
 
 {{range $m := .Messages}}
 
-## {{$m.Name}}
+## {{$m.Name}}
```

Comment:
these are tiny changes, that should be a separate commits; but I didn't want to make 2 commits, because changing the test scripts is so painful every time :/